Source/Engine/Editor/VQEditor.cpp

```diff
@@ -0,0 +1,253 @@
+#include "VQEditor.h"
+
+namespace VQEditor
+{
+	// GUI namespace begin:
+	// contains helper/wrapper functions for imgui
+
+	void GUI::Header(const char* title)
+	{
+		static const ImVec4 Orange{1.0f, 0.68f ,0.0f ,1.0f };
+		ImGui::GetFont()->FontSize += 3;
+		ImGui::TextColored(Orange, title);
+		ImGui::GetFont()->FontSize -= 3;
+	}
+
+	// void GUI::TextureField(const char* name, Texture*& texture)
```

Comment:
remove comments if no longer needed

Reply:
We will need Texture Field  when we create material inspector. Then I'll add todo comment for future.